### ISSUE TITLE
Spread operator in object literal causes SyntaxError

### ISSUE THREAD

**Issue by stevenbenisek** [rollup/rollup-plugin-commonjs#275]
Any module with a spread operator in an object literal will cause a SyntaxError.
This is because [Acorn](https://github.com/acornjs/acorn) doesn't support the spread operator in object literals (https://github.com/acornjs/acorn/issues/388) and all modules are parsed by [Acorn](https://github.com/acornjs/acorn) since https://github.com/rollup/rollup-plugin-commonjs/commit/b9191ae30d1fce2e11da1f014a7a01aa119a77b6. 
Before that the `checkFirstpass` function provided an early exit. 

Reduced test case:

*index.js*
```js
const obj = { foo: 'bar' };
const clone = { ...obj };
```

*rollup.config.js*
```js
import commonjs from 'rollup-plugin-commonjs';

export default {
  input: 'index.js',
  output: {
    format: 'cjs',
  },
  plugins: [commonjs()],
};
```
Yields `[!] (commonjs plugin) SyntaxError: Unexpected token`

**Comment by adrianheine** [comment]
This could be fixed with passing the parser instance to plugins as discussed in https://github.com/rollup/rollup/pull/1816#issuecomment-356946057.

**Comment by shannonmoeller** [comment]
Should be fixed by updating acorn!

acornjs/acorn#658
rollup/rollup#1613

**Comment by walterra** [comment]
I created a PR which could fix this: #294
